Library/Homebrew/formula_installer.rb

```diff
@@ -149,6 +149,8 @@ def install_bottle_for?(dep, build)
   def prelude
     Tab.clear_cache
     verify_deps_exist unless ignore_deps?
+    Homebrew.forbidden_license_check(formula) unless ENV["HOMEBREW_FORBIDDEN_LICENSES"].blank?
```

Comment:
Could maybe call this unconditionally and just make it a no-op if `ENV["HOMEBREW_FORBIDDEN_LICENSES"].blank?`